CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Changes in 2.0.0-beta.1:
+* Swashbuckle.AspNetCore updated to version >= 4.0.0
+* Schema Minimum and Maximum now support doubles (was only int)
+* WARNING: ScopedSwaggerMiddleware doesnot work as expected because Swashbuckle.AspNetCore changed a lot. Looking for workaround.
```

Comment:
Why did you close this issue? https://github.com/micro-elements/MicroElements.Swashbuckle.FluentValidation/issues/15

I have not tested the new beta build, but does this comment mean my app will crash if I use scope dependencies in my validators still without using any of the workarounds?

Reply:
Swashbuckle.AspNetCore has many changes and I suppose that it has bug. ScopedSwaggerMiddleware should be working but it does not work. There are some problems with DI objects instantiation in Swashbuckle.AspNetCore (dependencies declared as transient but resolves as singletons or caches).
I will debug this problem this week. I think DI problem with v4.0 should be declared as new issue.

Reply:
I went ahead and created the new issue for you.